src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectModifier.cs

```diff
@@ -236,77 +244,84 @@ internal async Task ModifyCsFile(CodeFile file, CodeAnalysis.Project project, Co
         /// <param name="options"></param>
         /// <param name="file"></param>
         /// <returns>modified root if there are changes, else null</returns>
-        private static CompilationUnitSyntax? ModifyRoot(DocumentBuilder documentBuilder, CodeChangeOptions options, CodeFile file)
+        private static SyntaxNode? ModifyRoot(DocumentBuilder documentBuilder, CodeChangeOptions options, CodeFile file)
```

Comment:
DocumentBuilder.ApplyChangesToMethod() returns a SyntaxNode, because it could be used for global methods or class methods, I can cast it back but it didn't seem necessary?

Reply:
That seems good nw.